internal/services/formatters/generic/trivy/formatter.go

```diff
@@ -161,9 +165,32 @@ func (f *Formatter) setVulnerabilities(cmd string, result *entities.Result, path
 	}
 }
 
+func findLineByFilename(target string, pkgName string, installedVersion string) string {
+	line := 0
+	file, err := os.Open(target)
+	if err != nil {
+		return strconv.Itoa(line)
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		if strings.HasPrefix(scanner.Text(), pkgName+" v"+installedVersion) {
+			return strconv.Itoa(line)
+		}
+		line++
+	}
+
+	if err := scanner.Err(); err != nil {
+		line = 0
+	}
+	return strconv.Itoa(line)
+}
+
```

Comment:
We already have a very similar logic on [getDependencyInfo](https://github.com/ZupIT/horusec/blob/main/internal/utils/file/file.go#L274).
I think that we can create a new function on `utils/`file` package that use this function to avoid duplicate the code.

Reply:
Hi, @matheusalcantarazup thanks for your advice. I have replaced the previous function with GetDependencyInfo.